AS_920_923_TTN_VN.yml

```diff
@@ -0,0 +1,93 @@
+band-id: AS_923
+sub-bands:
+- min-frequency: 920000000
+  max-frequency: 923000000
+duty-cycle: 0.01             # Duty cycle for this sub-band (optional; default: 1)
+max-eirp: 16
+uplink-channels:
+- frequency: 921400000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 921600000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 921800000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 922000000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 922200000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
```

Comment:
I think this should be `1`

Reply:
Because i want buget for fsk in future, if 1 value is than necessary, please change it for me

Reply:
This needs to be `1`. A concentrator radio can demodulate 4 frequencies, and you already define 4 frequencies for `0`. This has nothing to do with FSK.

Reply:
FTR: I don't think that's correct, @johanstokking. In our AS_923_925 we also have 5 channels on one radio:

![AS_923_925 yml](https://user-images.githubusercontent.com/181308/113149440-4c898380-9233-11eb-9da5-e7dee66c91b9.png)

If we want to add Std and FSK channels in the future, we can move the center frequencies of the radios around, and re-assign the channels.